ci/e2e/config-dispatch.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e +x -u
+
+export NODE_IP=$(cat cluster/metadata | jq -r '.nodeIP')
+if [ -e properties/keyval.properties ]; then
+    export IMAGE_TAG=$(cat properties/keyval.properties | sed 's/tag=//')
+fi
+export DOCKER_REGISTRY_HOST=$(cat cluster/metadata | jq -r '.registryHost')
+export DOCKER_REGISTRY_USER=$(cat cluster/metadata | jq -r '.registryUser')
+export DOCKER_REGISTRY_PASS=$(cat cluster/metadata | jq -r '.registryPass')
+export DOCKER_REGISTRY_EMAIL=$(cat cluster/metadata | jq -r '.registryEmail')
+
+echo "${NODE_IP} dev.dispatch.vmware.com api.dev.dispatch.vmware.com" >> /etc/hosts
+
+cat << EOF > config.yaml
```

Comment:
This file can be output via `dispatch install -d <path to config.yaml>`.  That will write the config file

Reply:
This is the installation input config, did you mean the CLI config stored by default in .dispatch, and which is created in `run-tests` step? if so, there are two problems with it:
* passing data between jobs in Concourse is surprisingly difficult, you need an external resource (like S3) to store file. It's just easier to create this file from the script.
* install does not set "skipAuth" flag, which is needed here.

Reply:
It's fine we can add skipAuth later if we need.  That sucks about not being able to pass artifacts in concourse.